docker/candlepin-base/cp-test

```diff
@@ -183,18 +187,24 @@ if [ "$LINTER" == "1" ]; then
     cd $CP_HOME
     echo "Running linter..."
 
-    tee /var/log/candlepin/checkstyle.log < /tmp/teepipe &
-    buildr checkstyle > /tmp/teepipe 2>&1
+    tee /var/log/candlepin/lint.log < /tmp/teepipe &
+    buildr lint > /tmp/teepipe 2>&1
 fi
 
 # TODO: keep track of return code?
 cd "$CP_HOME/$PROJECT"
 
-if [ "$UNITTEST" == "1" ]; then
-    echo "Running unit tests..."
-
-    tee /var/log/candlepin/unit_tests.log < /tmp/teepipe &
-    buildr test > /tmp/teepipe 2>&1
+if [ "$UNITTEST" -gt 0 ]; then
+    echo "Running unit tests $UNITTEST time(s)"
+    # run $UNITTEST time(s) to increase chance of capturing
+    # non deterministic unit test failures.
+    for (( i=1; i<=$UNITTEST; i++ ))
+    do 
+        tee /var/log/candlepin/unit_tests.log < /tmp/teepipe &
```

Comment:
Sorry, keep finding little things here.

The teepipe doesn't need to be setup every time, so we can probably move this out of the loop. Like the buildr comment, maybe we should change all the echos around so they show up in their respective logs.